demos-europe/demosplan-core
# T30866 administer imprint and privacy settings as customer master user

**Ticket:** https://yaits.demos-deutschland.de/T30866

Instead of the administrations of an organisation, the customer master user should do able to edit the imprint text and data protection text of each planning organisation.
An organisation is a "planning" organisation in case of it has one of three orgaTypes (MUNICIPALITY, PLANNING_AGENCY or HEARING_AUTHORITY_AGENCY) within the current customer.
The customer master user is managing the organisation via a special organisation list view, which has to be enhanced to allow showing the required text fields for each planning organisation.
To do so, the OrgaResourceType was enhanced to provide the necessary data along with the logic to update these organisation to allow edit the imprint and data protection text.


### How to review/test
Login as customer master user and navigate to the list of organisation. On each organisation which is a planning organisation (in the current customer), it should be possible to edit imprint and data protection text.
These texts should be displayed in the public area (no login requiered).

### Tasks
The check if an organisation is a planning organisation should probably be covered by a own ResourceType instead of special handling of existing "normal" OrgaResourceType. This has to be modeled and clarified and should result in another Story later on. 

Delete the checkbox if it doesn't apply/isn't necessary.
